megalista_dataflow/uploaders/display_video/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DV_API_VERSION = 'v1'
```

Comment:
Hi Pedro!
There is a new version of the DV360 API (v2), do you know if this endpoint is not going to change a lot in V2? Otherwise we would need to migrate soon.

Reply:
Hi, Ana!

Regarding to the endpoints we consume on this implementation, there won't be any significant changes. The v2 Alpha version will be released in Q2/Q3.